test/unit/Assessment/processFraud.js

```diff
@@ -779,4 +779,202 @@ describe('processFraud', function () {
   it.skip('consumes less gas to process than the summed fees of the fraudulent voting transactions', async function () {
     // [todo] Move this to integration tests instead
   });
+
+  it('reverts if system is paused', async function () {
+    const { assessment, master } = this.contracts;
+    const [fraudulentMember] = this.accounts.members;
+
+    await master.setEmergencyPause(true);
+
+    await expect(
+      assessment.processFraud(
+        0, // Index of the merkle tree root hash
+        [],
+        fraudulentMember.address, // The address of the fraudulent assessor
+        1, // The index of the last vote that is considered to be fraudulent
+        parseEther('100'), // The amount of stake to be burned
+        0, // The count of previous fraud attempts by this assessor
+        100, // Maximum iterations per tx
+      ),
+    ).to.be.revertedWith('System is paused');
+  });
+
+  it('allows to set voteBatchSize to 0', async function () {
+    const { assessment, individualClaims } = this.contracts;
+    const governance = this.accounts.governanceContracts[0];
+    const [fraudulentMember] = this.accounts.members;
+
+    await assessment.connect(fraudulentMember).stake(parseEther('100'));
+
+    await individualClaims.submitClaim(0, 0, parseEther('100'), '');
+
+    await assessment.connect(fraudulentMember).castVotes([0], [true], 0);
+
+    const burnAmount = parseEther('33');
+    await submitFraud({
+      assessment,
+      signer: governance,
+      addresses: [fraudulentMember.address],
+      amounts: [burnAmount],
+    });
+
+    {
+      const stake = await assessment.stakeOf(fraudulentMember.address);
+      expect(stake.amount).to.be.equal(parseEther('100'));
+    }
+
+    {
+      const fraudulentAssessment = await assessment.assessments(0);
+      expect(fraudulentAssessment.poll.accepted).to.be.equal(parseEther('100'));
+    }
+
+    await assessment.processFraud(
+      0, // Index of the merkle tree root hash
+      [],
+      fraudulentMember.address, // The address of the fraudulent assessor
+      0, // The index of the last vote that is considered to be fraudulent
+      burnAmount, // The amount of stake to be burned
+      0, // The count of previous fraud attempts by this assessor
+      0, // Maximum iterations per tx
```

Comment:
let's make a name constant here voteBatchSize = 0

because this is what the test is about, for clarity. took me some time to figure out what's the voteBatchSize value.